examples/kafka/start.sh

```diff
@@ -0,0 +1,51 @@
+#startup zookeeper
+cd /root/zookeeper-3.4.12/conf
+sed -i '14d' zoo.cfg
+sed -i "13a clientPort=$PAI_CONTAINER_HOST_zookeeper_PORT_LIST" zoo.cfg
+cd /root/zookeeper-3.4.12/bin
+./zkServer.sh start
+sleep 5s
```

Comment:
Why sleep for 5s?

Reply:
To make sure that zookeeper server has been started.